int_experian/int_experian_sgjc/cartridge/static/default/js/edqUtils.js

```diff
@@ -322,10 +322,16 @@ for (i = 0; i < buttonSelector.length; i++) {
 if (edqPhoneLineSelectors) { edqPhoneLineSelectors.addEventListener("mouseover", function() {enableButtonDisable(edqCurrentSubmitButtonSelector, false);}); }
 if (edqEmailLineSelector) { edqEmailLineSelector.addEventListener("mouseover", function() {enableButtonDisable(edqCurrentSubmitButtonSelector, false);}); }
 if (edqCurrentSubmitButtonSelector) { edqCurrentSubmitButtonSelector.addEventListener("mouseover", edqVerificationCallback); }
-if (document.querySelector("#form-submit")) { document.querySelector("#form-submit").addEventListener("mouseover", edqVerificationCallback); }
 function edqVerificationCallback() {
-	if ((edqEmailEnable) && (edqEmailLineSelector)) { edqPhoneEmailValidationCallback(edqValidateEmail, edqEmailLineSelector); }
-	if ((edqPhoneEnable) && (edqPhoneLineSelectors)) { edqPhoneEmailValidationCallback(edqValidatePhone, edqPhoneLineSelectors); }
+	/** Email Validation not restricting access.
```

Comment:
You need to restructure your code such that this function does not care about email or phone. This is a callback for verification, so it should only be concerned about address.